tee-worker/service/src/running-mode-config.json

```diff
@@ -43,8 +43,8 @@
         "discord_official_url": "https://discordapp.com",
         "discord_litentry_url": "http://54.255.182.249:9527",
         "discord_auth_token": "ABCDEFGHIJKLMNOPQRSTUVWXYZ",
-        "graphql_url": "https://graph.tdf-labs.io/",
-        "graphql_auth_key": "ac2115ec-e327-4862-84c5-f25b6b7d4533",
+        "achainable_url": "https://graph.tdf-labs.io/",
+        "achainable_auth_key": "ac2115ec-e327-4862-84c5-f25b6b7d4533",
```

Comment:
Is this the real value used in staging env or just placeholder?
Maybe we will need to update deployment env as well.

Reply:
They are just placeholders. Real value is only stored on our server.

Reply:
@BillyWooo do we need to make configuration changes on server before deploying this change ?